docs/api/rest/dicomweb-stow.md

```diff
@@ -0,0 +1,85 @@
+<!--
+SPDX-FileCopyrightText: © 2022 MONAI Consortium
+SPDX-License-Identifier: Apache License 2.0
+-->
+
+# DICOMWeb STOW-RS APIs
+
+The `dicomweb/` endpoint implements the specifications defined in section [6.6 STOW-RS Request/Response](https://dicom.nema.org/dicom/2013/output/chtml/part18/sect_6.6.html#sect_6.6.1.3.2.1.1) defined by the DICOM committee to provide the [DICOMWeb STOW-RS](https://www.dicomstandard.org/using/dicomweb/store-stow-rs) interface to enable the triggering of new workflows. 
+
+
+The *STOW-RS* service provides the following two endpoints:
+
+## POST /dicomweb/studies/[{study-instance-uid}]
+
+Triggers a new workflow request with the uploaded DICOM dataset.
+
+> [!IMPORTANT]
+> Each HTTP POST request triggers a new workflow request; the service *does not* support waiting for additional instances like the DIMSE service.
+
+### Parameters
+
+#### Query Parameters:
```

Comment:
This is not a query param as far as HTTP protocol is concern. It is an optional URI fragment to specify the resource. It would better to just cite what's in the standards, https://dicom.nema.org/dicom/2013/output/chtml/part18/sect_6.6.html#sect_6.6.1.1, which also says `instances not matching the StudyInstanceUID shall be rejected`

Reply:
Changed it to parameters.